sdv/constraints/tabular.py

```diff
@@ -1028,11 +1029,18 @@ def _sample_constraint_columns(self, table_data):
                         proposed_table_data[column] = 1.0
 
             else:
+                should_transform = True
+                conditions = sub_table_data[condition_columns]
+                transformed_conditions = self._hyper_transformer.transform(conditions)
```

Comment:
It seems like this introduces a `transform` that was not originally done, so the behavior is being changed. Why is this done?

Reply:
It should have been transformed before. It wasn't because the values are numeric and the column names were the same before so it didn't matter. Now that the column names are different, the model doesn't know the conditions' original names, and they should be transformed.